recipes/dandischema/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "dandischema" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2a1757510b811e49a392da08d96d397ab3137dbab2cd5e09905659ffea695491
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    - jsonschema
+    - pydantic[email] >=1.8.1
+    - ruamel.yaml
+    - typing_extensions  # [py37]
+
+test:
+  requires:
+    - pytest
+    - pytest-cov
+    - requests
+  source_files:
+    - tox.ini
```

Comment:
`tox.ini` isn't included in the PyPI tarball.

Reply:
This will be fixed in the next version.

Reply:
doh -- thought to suggest and accept but because I am not the owner -- can't... anyways 0.1.1 with @jwodder fixes is out and contains tox.ini
```
$> tar -xzvf dandischema-0.1.1.tar.gz | grep tox
dandischema-0.1.1/tox.ini
```
so green light